web/styles/zulip.css

```diff
@@ -1832,15 +1838,29 @@ div.focused_table {
 }
 
 .message_length_controller {
-    display: none;
-    text-align: center;
-    color: hsl(200deg 100% 40%);
+    .message_length_toggle {
+        display: none;
+        width: 100%;
+        height: 24px;
+        margin-bottom: 4px;
+        color: hsl(240deg 52% 53%);
+        background-color: hsl(240deg 44% 56% / 8%);
+        border-radius: 4px;
+        border: none;
+        outline: none;
+        font-variant: all-small-caps;
+        transition: all 150ms ease-out;
 
-    /* to make message-uncollapse easier */
-    margin-top: 10px;
+        &:hover {
+            color: hsl(240deg 52% 53%);
+            background-color: hsl(240deg 44% 56% / 15%);
+        }
 
-    &:hover {
-        text-decoration: underline;
+        &:active {
+            color: hsl(240deg 52% 53);
```

Comment:
This is missing a `%`.  We should make `color` and `background-color` a varibale. This would also removed dependency on `dark_theme`.

Reply:
Great, added color variables :)